turbo/generators/config.ts

```diff
@@ -1,6 +1,31 @@
-module.exports = function (plop) {
-  const toFileName = (str) => str.toLowerCase().replace(/ /g, '-')
-  plop.setHelper('toFileName', toFileName)
+import type { PlopTypes } from '@turbo/gen'
+import path from 'path'
+import * as helpers from './helpers'
+
+interface TestResponse extends Parameters<PlopTypes.DynamicActionsFunction> {
+  appDir: string
+  type: 'e2e' | 'production' | 'development' | 'unit'
+}
+
+interface ErrorResponse extends Parameters<PlopTypes.DynamicActionsFunction> {
+  name: string
+  title: string
+  why: string
+  fix: string
+}
+
+function validateNonEmptyString(field: string) {
+  return function (value: string) {
+    if (/.+/.test(value)) {
+      return true
+    }
+    return `${field} is required`
+  }
+}
+
+export default function generator(plop: PlopTypes.NodePlopAPI): void {
+  // make our custom helpers available for use in templates as handlebars helpers
+  helpers.init(plop)
```

Comment:
In the future, it might be nifty to loop over the helpers here.

Although not necessary now since there is only one.